app/src/commonMain/kotlin/com/popovanton0/blueprint/app/Lab.kt

```diff
@@ -407,19 +397,19 @@ private fun ButtonDemo2() = Box {
                         text = "Lorem ".repeat(1),
                         fontSize = textSize.value.sp,
                         style = LocalTextStyle.current.copy(
-                            platformStyle = PlatformTextStyle(
-                                includeFontPadding = false
-                            )
+//                            platformStyle = PlatformTextStyle(
```

Comment:
I think all comments with `PlatformTextStyle` can be removed, because `includeFontPadding = false` has become the default for some time now. I've seen a few comments in other files